src/index.tsx

```diff
@@ -1,14 +1,35 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route, RouterProvider, createHashRouter, createRoutesFromElements,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { App } from './App';
+import { HomePage } from './routes/HomePage';
+import { PeoplePage } from './routes/PeoplePage';
+import { NotFound } from './routes/NotFound';
+
+const router = createHashRouter(
+  createRoutesFromElements(
```

Comment:
let's not test the cognitive abilities of the developers🥲

idk why we need such complicated structures, imo, we can put it directly in render method without any additional methods

Reply:
This is just another way to define router. [Picking a Router](https://reactrouter.com/en/main/routers/picking-a-router) It's for supporting new React Router features such as loader, errorElement, etc,

Reply:
In this case it unnecessary but anyway :)